app/views/import_batches/preview.html.erb

```diff
@@ -27,7 +27,12 @@
   <h4 class="add-bottom-margin">Running this import will</h4>
   <ul class="list-group">
     <li class="list-group-item">Create <%= pluralize(@preview.redirect_count, 'new redirect') %></li>
-    <li class="list-group-item">Create <%= pluralize(@preview.archive_count, 'new archive') %></li>
+    <li class="list-group-item">
+      Create <%= pluralize(@preview.archive_count, 'new archive') %>
+      <% if @preview.custom_archive_url_count > 0 %>
+        (<%= pluralize(@preview.custom_archive_url_count, 'with custom URL') %>)
```

Comment:
I came across that view while working on this, but I found it hard to understand. Obviously it's wrong to introduce inconsistent wording, but are we sure that the original wording says what it means?